src/CoreWCF.Primitives/src/CoreWCF/Runtime/BufferedOutputStream.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace CoreWCF.Runtime
 {
-    internal class BufferedOutputStream : Stream
+    public class BufferedOutputStream : Stream
     {
```

Comment:
Can usage of this be replaced with System.IO.BufferedStream?

Reply:
This was addressed in 53a8acf9b0576969179c65e5a97cf9dc431de9fe.